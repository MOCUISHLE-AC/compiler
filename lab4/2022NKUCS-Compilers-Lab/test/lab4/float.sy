int main(){
    float a=0.114514;
}