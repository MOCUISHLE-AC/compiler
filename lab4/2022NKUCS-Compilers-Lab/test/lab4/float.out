[DEBUG LAB4]: 	INT 	 int	        1 	        0
[DEBUG LAB4]: 	MAIN	 main 	        1	        4
[DEBUG LAB4]: 	LPAREN	(	        1	        8
[DEBUG LAB4]: 	RPAREN	)	        1	        9
[DEBUG LAB4]: 	LBRACE	{ 	        1	       10
[DEBUG LAB4]: 	FLOAT 	 float	        2 	        4
[DEBUG LAB4]: 	ID: 	 a 	        2	       10 	 0x55c172b390a0
[DEBUG LAB4]: 	ASSIGN	 = 	        2 	       11
[DEBUG LAB4]: 	FLOAT:	0.114514 	       2 	       12
[DEBUG LAB4]: 	SEMICOLON	 ;	        2	       20
[DEBUG LAB4]: 	RBRACE	 } 	        3	        0
